dotnet/src/webdriver/Firefox/FirefoxProfile.cs

```diff
@@ -114,8 +114,14 @@ public static FirefoxProfile FromBase64String(string base64)
         /// Adds a Firefox Extension to this profile
         /// </summary>
         /// <param name="extensionToInstall">The path to the new extension</param>
+        /// <exception cref="ArgumentNullException">If <paramref name="extensionToInstall"/> is <see langword="null"/>.</exception>
         public void AddExtension(string extensionToInstall)
         {
+            if (extensionToInstall is null)
```

Comment:
This already throws `ArgumentNullException` because of the `Dictionary.Add` just below